pkg/cli/admin/release/extract_tools.go

```diff
@@ -222,10 +233,13 @@ func (o *ExtractOptions) extractCommand(command string) error {
 		}
 	} else {
 		willArchive = true
-		targets = availableTargets
-		for i := range targets {
-			targets[i].AsArchive = true
-			targets[i].AsZip = targets[i].OS == "windows"
+		// Only non-Hidden targets will be included in an archive.
+		for _, target := range availableTargets {
```

Comment:
The ideal outcome is:

oc adm release extract —tools

Doesn’t include it, but

oc adm release extract —command openshift-baremetal-installer

Extracts the binary

oc adm release extract —command openshift-baremetal-installer —command-os=*

Creates an archive.

Hence the suggestion for optional - the explicit request overrides optional, rather than the current behavior here with always excluding from archive.